File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        j = i
        while(i < len(nums) and j <len(nums)):
            while(j<len(nums)):
                if(nums[j] != nums[i]):
                    i+= 1
                    nums[i] = nums[j]
                
                j+= 1
        return i+1

File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.go
import ("fmt")

func removeDuplicates(nums []int) int {
    left := 0
    size := len(nums)
    current := nums[left]
    right := left + 1
    count := 1
    for right < size {
        if nums[right] != current {
            nums[left+1] = nums[right]
            count += 1
            left += 1
            current = nums[right]
        }
        right += 1
    }
    fmt.Println(nums)
    return count
    
}